README.md

```diff
@@ -14,30 +23,28 @@ The eva package can be installed using pip:
 
 ### Usage
 
-eva uses a strict dictionary only API. This ensures flexible use for different applications. The most straightforward use is achieved by choosing the Class containing the diagnostic and passing a yaml configuration file:
-
-	eva ObsCorrelationScatter obs_correlation.yaml
-
-Alternatively you can invoke eva passing the class name within the yaml configuration:
+eva uses a strict dictionary/configuration only API. This ensures flexible use for different applications. The most straightforward use of eva is achieved by passing it a yaml configuration file on the command line:
 
 	eva obs_correlation.yaml
 
-Where the yaml must contain a list of the diagnostics to be used:
+Where the yaml must contain a list of the diagnostics to be used in the following format:
 
 ```
-applications:
-  - application name: ObsCorrelationScatter
+diagnostics:
+  - diagnostic name: ObsCorrelationScatter
     ...
-  - application name: ObsMapScatter
+  - diagnostic name: ObsMapScatter
     ...
 ```
 
-eva can also be invoked from another Python module that passes a dictionary, rather than a Yaml file that is later translated into a dictionary. This is achieved as follows:
+eva can also be invoked from another Python module that passes it a dictionary, rather than a Yaml file that is later translated into a dictionary. This is achieved as follows:
```

Comment:
be consistent here, `yaml` and `Yaml` both in this section, which is best? Or is it `YAML`?